app/src/main/java/uk/gov/android/authentication/integrity/keymanager/ECKeyManager.kt

```diff
@@ -56,8 +57,10 @@ class ECKeyManager : KeyStoreManager {
             update(input)
             sign()
         }
-        val verifyResult = verify(input, signature)
-        Log.d("VerifySignature", "$verifyResult")
+        if (BuildConfig.DEBUG) {
```

Comment:
I had to wrap the Log.d above in a `if (BuildConfig.DEBUG)` weird that the linter only complains about this one for you.

Reply:
Yeah, weird this. This particular section will likely be removed soon anyway